.github/pull_request_template.md

```diff
@@ -1,5 +1,7 @@
 Closes #
 
+
+[ ] Check if this PR introduces [backward-incompatible DB changes](../docs/deployment#django-migrations-is-my-deploy-safe)
```

Comment:
Can we add a `  - [ ] Turn the issue this is a PR for into a tracking issue, and make sure there are 2 or 3 separate PRs to handle the changes` here, too?

Reply:
I've added a "related issue/PR" field, as it's more versatile. I don't expect the majority of our PRs to introduce backward-incompatible migrations.